apps/.flowconfig

```diff
@@ -0,0 +1,15 @@
+[ignore]
+.*/node_modules/fbjs/.*
+.*/node_modules/react-color/.*
+.*/node_modules/radium/.*
+.*/node_modules/editions/.*
+.*/node_modules/cjson/.*
```

Comment:
Because flowtype will actually enforce types that it can infer from third party modules. These particular modules have some funny bugs in them that cause flowtype to complain erroneously. There are issues out to fix them already, either in the flowtype project, or in the other projects being ignored.